app/views/filters/_form.html.erb

```diff
@@ -52,14 +52,17 @@
       <div id="granular_form" style="<%= f.object.granular? ? '' : 'display:none' %>">
         <%= checkbox_f f, :unlimited?,
                        :label_help => (_("If the unlimited field is enabled, the filter applies to all resources of the selected type. If the unlimited </br> field is disabled, you can specify further filtering using Foreman search syntax in the search field.</br> If the role is associated with %s, the filters are not considered unlimited</br> as they are scoped accordingly.") % org_loc_string('or')).html_safe %>
-
-
-        <%= autocomplete_f f, :search,
```

Comment:
can we simply replace the helper? so here (and any other place that uses it will auto update?)

Reply:
the `autocomplete_f` was being called only here so I think we could remove the place it is defined too and move to use `mount_react_component('FormAutocomplete'...` instead

Reply:
keep in mind plugins https://github.com/theforeman/foreman_discovery/blob/develop/app/views/discovery_rules/_form.html.erb#L18 :-)

changing internal implementation of this helper would be nice if it's 100% compatible

Reply:
thanks @ares, Ohad and I have discussed about it offline,
so I will just change the inner implementation of `autocomplete_f`

Reply:
Done :)